src/js/mermaid.js

```diff
@@ -23,6 +23,12 @@ document.addEventListener("DOMContentLoaded", function (event) {
         theme: theme,
         themeVariables: {
           darkMode: darkMode
+        },
+        mermaid: {
+          // Fix height of mermaid SVG elements (see https://github.com/mermaid-js/mermaid/issues/2481)
+          callback: (id) => {
+            document.getElementById(id).setAttribute("height", "100%")
+          }
```

Comment:
Should I move this change to a separate PR?

Reply:
Yes, please create a dedicated PR, make it easier in case we need to revert it. Wondering why this was not fixed upstream yet if a fix would be that simple :thinking: